mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java

```diff
@@ -51,6 +52,25 @@ public ChangeManagerImpl(Vertx vertx, String tenantId) { //NOSONAR
     this.tenantId = TenantTool.calculateTenantId(tenantId);
   }
 
+
+  @Override
+  public void deleteChangeManagerJobExecutions(DeleteJobExecutionsDto entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(c -> {
+      try {
+        jobExecutionService.deleteJobExecutionsByIds(entity.getIds(), tenantId)
+          .map(optionalJobExecution -> !optionalJobExecution.booleanValue()?
+            new BadRequestException(format("JobExecution with id '%s' was not found", entity.getIds())): optionalJobExecution)
+          .map(DeleteChangeManagerJobExecutionsResponse::respond200WithApplicationJson)
+          .map(Response.class::cast)
+          .otherwise(ExceptionHelper::mapExceptionToResponse)
+          .onComplete(asyncResultHandler);
+      } catch (Exception e) {
+        LOGGER.error("Failed to delete JobExecutions by ids {} : Exception : ", entity.getIds(), e);
```

Comment:
"Exception :" is not needed here. It is not obvious from the api of LOGGER.error that the error message is appended at the end.

Reply:
Hi @okolawole-ebsco - Updated changes as per review comment, Thanks 🙂